src/js/feedback/components/FeedbackForm.jsx

```diff
@@ -54,30 +64,43 @@ class FeedbackForm extends React.Component {
     return !this.state.suppressEmailErrors ? this.props.formErrors.email : '';
   }
 
+  responseErrorMessage = () => {
+    return !this.state.suppressResponseErrors ? this.props.formErrors.shouldSendResponse : '';
+  }
+
   render() {
+    const { shouldSendResponse } = this.props.formValues;
     return (
-      <form id="feedback-form" className="feedback-form" onSubmit={this.sendFeedback}>
-        <h4 className="feedback-widget-title">Tell us what you think</h4>
+      <form id="feedback-form" className="feedback-form" onSubmit={this.onSubmit}>
         <div className="va-flex">
           <div className="feedback-widget-form-container">
             <div className="feedback-widget-desc-container">
               <ErrorableTextarea
-                label="What can we do to make Vets.gov better?"
+                label="Tell us about your ideas to make Vets.gov better."
                 name="description"
                 onValueChange={this.setDescription}
                 errorMessage={this.descriptionErrorMessage()}
                 field={{ value: this.props.formValues.description, dirty: false }}
                 ref={component => { this.descriptionComp = component; }}
                 required/>
             </div>
-            <ErrorableCheckbox
-              name="should-send-response"
-              label="I would like to receive a response about my feedback."
-              checked={this.props.formValues.shouldSendResponse}
-              onValueChange={(shouldSendResponse) => this.props.setFormValues({ shouldSendResponse })}/>
+
+            <ErrorableRadioButtons
+              name="shouldSendResponse"
+              id="shouldSendResponse"
+              label="Would you like us to follow up with you about your ideas?"
+              options={[
+                { label: 'Yes', value: 'yes' },
+                { label: 'No', value: 'no' }
+              ]}
+              onValueChange={this.setResponse}
+              errorMessage={this.responseErrorMessage()}
+              value={{ value: shouldSendResponse, dirty: false }}
+              required/>
```

Comment:
Should we include `ref` here since it is used on the other fields as well? Are these refs already being or planned to be used? @ncksllvn 
```
ref={component => { this.shouldSendResponseComp = component; }}
```